telemetry/data_feature_usage.go

```diff
@@ -279,3 +282,31 @@ func postReportNonTransactionalCounter() {
 func getGlobalKillUsageInfo() bool {
 	return config.GetGlobalConfig().EnableGlobalKill
 }
+
+// TiFlashModeStatistics records the usage info of Fast Mode
+type TiFlashModeStatistics struct {
+	// fast mode table / (fast mode table + normal mode table) , these tables must have tiflash replica
+	FastModeTableProportion float64 `json:"fast_mode_table_proportion"`
```

Comment:
Prefer to just use raw data for larger data exploration potential, i.e. separated fields for number of fast mode tables, number of normal mode tables, number of all tables.